beeswax/openrtb/extension.proto

```diff
@@ -299,6 +299,9 @@ message DeviceExtensions {
   // GDPR-compliant hashed ipv6
   optional string ipv6_hashed = 12;
 
+  // Added by augmentor in Beeswax Front End Server (FES)
+  repeated string augmented_models = 13;
```

Comment:
Do you need to re-build the binary with this field?

Reply:
The binary is built with the new field.